hail/python/hailtop/hailctl/batch/wait.py

```diff
@@ -13,4 +17,4 @@ def main(args, pass_through_args, client):  # pylint: disable=unused-argument
         sys.exit(1)
 
     batch = maybe_batch
-    print(batch.wait())
+    print(json.dumps(batch.wait(disable_progress_bar=True)))
```

Comment:
Can you add a `-o` here as well? We should try to stay backwards compatible even though ~0 people are using this.